bakerydemo/settings/base.py

```diff
@@ -167,12 +283,88 @@
     os.path.join(PROJECT_DIR, "static"),
 ]
 
+STATICFILES_STORAGE = "whitenoise.storage.CompressedManifestStaticFilesStorage"
+
 STATIC_ROOT = os.path.join(PROJECT_DIR, "collect_static")
 STATIC_URL = "/static/"
 
 MEDIA_ROOT = os.path.join(PROJECT_DIR, "media")
 MEDIA_URL = "/media/"
 
+# AWS creds may be used for S3 and/or Elasticsearch
+AWS_ACCESS_KEY_ID = os.getenv("AWS_ACCESS_KEY_ID", "")
+AWS_SECRET_ACCESS_KEY = os.getenv("AWS_SECRET_ACCESS_KEY", "")
+AWS_REGION = os.getenv("AWS_REGION", "")
+
+# Simplified static file serving.
+# https://warehouse.python.org/project/whitenoise/
+
+# AWS S3 buckets configuration
+# This is media files storage backend configuration. S3 is our preferred file
+# storage solution.
+# To enable this storage backend we use django-storages package...
+# https://django-storages.readthedocs.io/en/latest/backends/amazon-S3.html
+# ...that uses AWS' boto3 library.
+# https://boto3.amazonaws.com/v1/documentation/api/latest/index.html
+#
+# Three required environment variables are:
+#  * AWS_STORAGE_BUCKET_NAME
+#  * AWS_ACCESS_KEY_ID
+#  * AWS_SECRET_ACCESS_KEY
+# The last two are picked up by boto3:
+# https://boto3.amazonaws.com/v1/documentation/api/latest/guide/configuration.html#environment-variables
+if "AWS_STORAGE_BUCKET_NAME" in os.environ:
+    # Add django-storages to the installed apps
+    INSTALLED_APPS = INSTALLED_APPS + ["storages"]
+
+    # https://docs.djangoproject.com/en/stable/ref/settings/#default-file-storage
+    DEFAULT_FILE_STORAGE = "storages.backends.s3boto3.S3Boto3Storage"
+
+    AWS_STORAGE_BUCKET_NAME = os.environ["AWS_STORAGE_BUCKET_NAME"]
+
+    # Disables signing of the S3 objects' URLs. When set to True it
+    # will append authorization querystring to each URL.
+    AWS_QUERYSTRING_AUTH = False
+
+    # Do not allow overriding files on S3 as per Wagtail docs recommendation:
+    # https://docs.wagtail.io/en/stable/advanced_topics/deploying.html#cloud-storage
+    # Not having this setting may have consequences in losing files.
+    AWS_S3_FILE_OVERWRITE = False
+
+    # Default ACL for new files should be "private" - not accessible to the
+    # public. Images should be made available to public via the bucket policy,
+    # where the documents should use wagtail-storages.
+    AWS_DEFAULT_ACL = "private"
+
+    # We generally use this setting in the production to put the S3 bucket
```

Comment:
Perhaps we should change some of this language to be more instructive rather than descriptive of how 'we' do things.